src/main/java/de/rytrox/gamestate/mixed/example/FifthPhase.java

```diff
@@ -0,0 +1,51 @@
+package de.rytrox.gamestate.mixed.example;
+
+import de.rytrox.gamestate.mixed.State;
+import de.rytrox.varo.Varo;
+import de.rytrox.varo.scoreboard.ScoreBoardManager;
+import de.rytrox.varo.teams.GameTimeService;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class FifthPhase extends State {
+
+    private final Varo main;
+
+    private final GameTimeService gameTimeService;
+    private final ScoreBoardManager scoreBoardManager;
+
+    public FifthPhase(@NotNull Varo main,
+                      @NotNull GameTimeService gameTimeService,
+                      @NotNull ScoreBoardManager scoreBoardManager) {
+        super("forth");
+
+        this.main = main;
+        this.gameTimeService = gameTimeService;
+        this.scoreBoardManager = scoreBoardManager;
+    }
+
+
+    /**
+     * Diese Methode möchte ich für die Vererbung offen lassen!
+     */
+    @Override
+    public void onEnable() {
+        // Da die beiden Listener schon registriert sind, brauch ich hier nichts tun. Yay!!
```

Comment:
yay, macht es ja total übersichtlich

Reply:
ich weiß, dass es in v2 auch nicht ideal war, den listener bei jedem state change neu zu enablen, allerdings hätte man das durch einen Vergleich beim Wechsel lösen können. Bei dir ist jetzt einfach nicht mehr auf einem Blick klar, welcher State, welche Listener hat. Das hat die Übersichtlichkeit definitiv gesteigert

Reply:
Wird btw. bei dir dann auch zu fehlverhalten führen, wenn ich nicht von einem Gamestate brav in den nächsten gehe, sondern vlt. einen überspringe

Reply:
Das Problem habe ich in der Tat, ich muss mir sicher sein, dass ich auf jeden Fall alles bedeutsame im onDisable disable, damit das alles reibungslos funktioniert

Reply:
Theoretisch kann ich den Listener aber auch neuregistrieren und im onDisable der Vierten Phase disablen